sklearn/decomposition/_online_lda.pyx

```diff
@@ -0,0 +1,73 @@
+cimport cython
+cimport numpy as np
+import numpy as np
+
+np.import_array()
+
+cdef np.float64_t NEGINF = -np.inf
+
+from libc.math cimport fabs
+from scipy.special import psi
+
+
+@cython.boundscheck(False)
+@cython.wraparound(False)
+def mean_change(np.ndarray[ndim=1, dtype=np.float64_t] arr_1,
```

Comment:
I guess you benchmarked it, what is the speedup of doing this vs ``np.abs(arr_1 - arr_2).mean()``? I guess you save the allocation of two temporary arrays? Not sure since when ``abs`` has an ``out`` and if that would make it better....